docs/Dockerfile

```diff
@@ -1,85 +1,29 @@
-FROM jupyter/base-notebook:latest
+FROM mcr.microsoft.com/dotnet/sdk:7.0
 
-# Install .NET CLI dependencies
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get -y install python3 python3-pip python3-dev ipython3
+
+RUN python3 -m pip install --no-cache-dir notebook jupyterlab
 
-ARG NB_USER=fsdocs-user
+ARG NB_USER=jovyan
```

Comment:
Any reason to have this specific user name?

Reply:
Sorry did not see it. It is just the default username that was used in the samples i ported this from. it should not matter at all

Reply:
yeah you are right, i'll leavit it as-is